stats/stats-proto/proto/api_config_http.yaml

```diff
@@ -0,0 +1,10 @@
+type: google.api.Service
+config_version: 3
+
+http:
+  rules:
+    - selector: blockscout.stats.v1.Stats.GetStats
+      get: /health
```

Comment:
typo?

Reply:
yeah, I've fixed it, but probably still will change it later